holoviews/core/data/dictionary.py

```diff
@@ -358,5 +377,22 @@ def iloc(cls, dataset, index):
             return arr if isscalar(arr) else arr[0]
         return new_data
 
+    @classmethod
+    def has_holes(cls, dataset):
+        from holoviews.element import Polygons
+        key = Polygons._hole_key
+        return key in dataset.data and isinstance(dataset.data[key], list)
+
+    @classmethod
+    def holes(cls, dataset):
+        from holoviews.element import Polygons
+        key = Polygons._hole_key
+        if key in dataset.data:
+            return [[[np.asarray(h) for h in hs] for hs in dataset.data[key]]]
+        else:
```

Comment:
Why can't you use the super definition of `holes` if holes aren't defined/supported?

Reply:
Unless the default implementation works for all interfaces except this one specifically?

Reply:
Probably can do that.